django_refugeerights/django_refugeerights/settings.py

```diff
@@ -78,7 +78,7 @@
 
 DATABASES = {
     'default': dj_database_url.config(
-        default='postgis://postgres:@localhost/django_refugeerights'),
+        default='postgis://trevisto:trevisto@localhost/refugeedb1'),
```

Comment:
We need to change this back for tests to be able to run on travis. You can add the postgres role to your local instance using `psql -U trevisto -c "create role postgres createdb;"`, and then create the db using `psql -U postgres -c "create database django_refugeerights;"`, then you will also need to create the extensions `psql -U postgres -d django_refugeerights -c "create extension xxx;"`, with the `postgis` and `postgis_topology` extensions.